driver/ppm_cputime.c

```diff
@@ -48,15 +49,26 @@ void ppm_task_cputime_adjusted(struct task_struct *p, cputime_t *ut, cputime_t *
 #endif
 
 #ifdef CONFIG_VIRT_CPU_ACCOUNTING_GEN
+
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 13, 0)) || (PPM_RHEL_RELEASE_CODE > 0 && PPM_RHEL_RELEASE_CODE >= PPM_RHEL_RELEASE_VERSION(7, 7))
```

Comment:
I added a comment to the PPM_* macros but I'm not sure what I can say here that isn't already expressed by the code. If we're building for >= 4.13 _or_ for RHEL >= 7.7, use the new macros, otherwise use the old ones.